opensrp-reveal/build.gradle

```diff
@@ -166,7 +166,12 @@ tasks.withType(Test) {
 
 dependencies {
 
-    implementation('io.ona.kujaku:library:0.8.0') {
+    implementation('io.ona.kujaku:library:0.8.1') {
+        exclude group: 'com.android.volley'
+        exclude group: 'kujaku', module : 'utils'
+    }
+
+    implementation('io.ona.kujaku:utils:0.8.1') {
```

Comment:
The version will probably remain as `0.8.0` and you can just use whatever the `library` has imported which will be the correct version

Reply:
This was needed because of deploying locally, it shall be removed when the library is officially published

Reply:
Cool